Ogury/src/main/java/com/mopub/mobileads/OguryConfigurationParser.java

```diff
@@ -0,0 +1,38 @@
+package com.mopub.mobileads;
+
+import android.text.TextUtils;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import java.util.Map;
+
+public class OguryConfigurationParser {
+
+    private static final String AD_UNIT_ID_KEY = "ad_unit_id";
+    private static final String ASSET_KEY_KEY = "asset_key";
+
+    @Nullable
+    public static String getAdUnitId(Map<String, String> adDataExtras) {
+        if (adDataExtras != null && !adDataExtras.isEmpty()) {
+            return adDataExtras.get(AD_UNIT_ID_KEY);
+        }
+        return null;
+    }
+
+    @Nullable
+    public static String getAssetKey(Map<String, String> adDataExtras) {
+        if (adDataExtras != null && !adDataExtras.isEmpty()) {
+            return adDataExtras.get(ASSET_KEY_KEY);
+        }
+        return null;
+    }
+
+    public static boolean isAssetKeyValid(@Nullable String assetKey) {
+        return assetKey != null && !TextUtils.isEmpty(assetKey);
+    }
+
+    public static boolean isAdUnitIdValid(@Nullable String adUnitId) {
+        return adUnitId != null && !TextUtils.isEmpty(adUnitId);
```

Comment:
**suggestion**: TextUtils.isEmpty(...) already check for nullability.
```suggestion
        return !TextUtils.isEmpty(adUnitId);
```

Reply:
Fixed in ecdb97582edd887a4ba69ae77ba6a6bd05a8d402.